tests/src/test/scala/cats/tests/XorTests.scala

```diff
@@ -12,7 +12,8 @@ import org.scalacheck.Arbitrary._
 import scala.util.Try
 
 class XorTests extends CatsSuite {
-  checkAll("Xor[String, Int]", GroupLaws[Xor[String, Int]].monoid)
+  checkAll("Monoid[Xor[String, Int]]", GroupLaws[Xor[String, Int]].monoid)
+  checkAll("Semigroup[Xor[String, NonEmptyList[Int]]]", GroupLaws[Xor[String, NonEmptyList[Int]]].semigroup)
```

Comment:
Minor: `monoid`/`semigroup` are already included in the test names, so I think the convention has been to not include `Monoid`/`Semigroup` in the string argument.